iron-menu-behavior.d.ts

```diff
@@ -0,0 +1,175 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   iron-menu-behavior.html
+ */
+
+/// <reference path="../polymer/types/polymer.d.ts" />
+/// <reference path="../iron-selector/iron-multi-selectable.d.ts" />
+/// <reference path="../iron-a11y-keys-behavior/iron-a11y-keys-behavior.d.ts" />
+
+declare namespace Polymer {
+
+  /**
+   * `Polymer.IronMenuBehavior` implements accessible menu behavior.
+   */
+  interface IronMenuBehavior extends Polymer.IronMultiSelectableBehavior, Polymer.IronA11yKeysBehavior {
+    keyBindings: object|null;
+
+    /**
+     * Returns the currently focused item.
+     */
+    readonly focusedItem: object|null;
+
+    /**
+     * The attribute to use on menu items to look up the item title. Typing the first
+     * letter of an item when the menu is open focuses that item. If unset, `textContent`
+     * will be used.
+     */
+    attrForItemTitle: string|null|undefined;
+    disabled: boolean|null|undefined;
```

Comment:
Default value is `false`, so I think this should be just `boolean`

Reply:
Done.